.eslintrc.js

```diff
@@ -1,28 +1,20 @@
 module.exports = {
-  extends: 'airbnb-base',
+  extends: ['airbnb-base', 'prettier'],
   env: {
-    'node': true
+    node: true
   },
   rules: {
     // Using 'console' is perfectly okay for a Node.js CLI tool and avoiding
     // it only brings unnecessary complexity
     'no-console': 'off',
 
-    // Node 6 does not support dangling commas in function arguments
-    'comma-dangle': [
-      'error',
-      {
-        'arrays': 'always-multiline',
-        'objects': 'always-multiline',
-        'functions': 'never'
-      }
-    ],
-
     // This is to allow a convention for exporting functions solely for
     // the purpose of the unit tests, see
     // https://github.com/apiaryio/dredd-transactions/pull/179#discussion_r206852270
     'no-underscore-dangle': 'off',
 
+    'import/prefer-default-export': 'off',
```

Comment:
Allows `export const a = 'anything'` exports from files.

- Makes internals easier to export for testing
- Doesn't pollute the default export